README.md

```diff
@@ -158,6 +158,43 @@ uv run scripts/serve_policy.py policy:checkpoint --policy.config=pi0_fast_libero
 
 This will spin up a server that listens on port 8000 and waits for observations to be sent to it. We can then run the Libero evaluation script to query the server. For instructions how to install Libero and run the evaluation script, see the [Libero README](examples/libero/README.md).
 
+If you want to embed a policy server call in your own robot runtime, below is a minimal example of how to do so (also check out the [Libero eval script](examples/libero/main.py) for a full example):
```

Comment:
I feel like this is too much detail for the main `README`. It will make it difficult for users with any other need to find what they're looking for.

Reply:
Moved it to the remove_inference documentation instead!